src/preprocessing/assertion_pipeline.cpp

```diff
@@ -186,6 +198,7 @@ void AssertionPipeline::enableProofs(smt::PreprocessProofGenerator* pppg)
   {
     d_andElimEpg.reset(
         new LazyCDProof(d_env, nullptr, userContext(), "AssertionsAndElim"));
+    d_rewpg.reset(new RewriteProofGenerator(d_env));
```

Comment:
shouldn't it be set independently from `d_andElimEpg`?

Reply:
These should never be used separately, but I suppose we could initialize them based on 2 guards.